packages/web3-providers/src/types.ts

```diff
@@ -291,7 +291,7 @@ export namespace NonFungibleTokenAPI {
 export namespace StorageAPI {
     export interface Storage {
         set<T extends {}>(key: string, value: T): Promise<void>
-        get<T>(key: string): Promise<T | void>
+        get<T>(key: string): Promise<T | undefined>
         delete?: (key: string) => Promise<void>
```

Comment:
```suggestion
        delete?(key: string): Promise<void>
```

Reply:
Resolve this here https://github.com/DimensionDev/Maskbook/pull/5462.